SpatialGDK/Source/SpatialGDKTests/SpatialGDKServices/LocalDeploymentManager/LocalDeploymentManagerUtilities.cpp

```diff
@@ -60,7 +60,7 @@ bool FStartDeployment::Update()
 		const FString LaunchConfig =
 			FPaths::Combine(FPaths::ConvertRelativePathToFull(FPaths::ProjectIntermediateDir()), AutomationLaunchConfig);
 		const FString LaunchFlags = SpatialGDKSettings->GetSpatialOSCommandLineLaunchFlags();
-		const FString SnapshotName = SpatialGDKSettings->GetSpatialOSSnapshotToLoad();
+		const FString SnapshotName = SpatialGDKSettings->GetSpatialOSSnapshotToLoadPath();
```

Comment:
With preview-4 comes the ability to test correctly. This means switching from using a snapshot name, which was expected with spatiald, to snapshot path, which is what's expected with standalone